test/node/int_32_tests.js

```diff
@@ -17,5 +19,49 @@ describe('Int32', function() {
       expect(new Int32(new Number(value)).valueOf()).to.equal(value);
       done();
     });
+
+    it('Hex', function(done) {
+      expect(new Int32(hexValue).valueOf()).to.equal(value);
+      done();
+    });
+
+    it('Octal', function(done) {
+      expect(new Int32(octalValue).valueOf()).to.equal(value);
+      done();
+    });
+
+    it('Cases covering Int32 value conversion using +', function(done) {
```

Comment:
@nbbeeken I've added all the tests we discussed here, let me know if I forgot any, ran this before and after PR and all work.

Reply:
Coverage here looks good 👍 🚀